app/models/support_interface/cancel_course_form.rb

```diff
@@ -0,0 +1,16 @@
+module SupportInterface
+  class CancelCourseForm
```

Comment:
If it's the last course, should this also notify referees their responses are no longer needed?

Reply:
No, that's why this PR doesn't allow cancellation of the last course (https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1884/files#diff-84183967e9402039054b942451a960d4R8-R10). Perhaps I'll combine that feature in this PR as well, so we can look at it as a single flow.

Reply:
Ah, great. I wonder if it's a separate thing, then, as it might need to be invoked from the candidate side as well if they withdraw from the last course?

Reply:
I've added the feature to cancel the entire application in https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1884/commits/e085fc88b96d68bc1c127f1012aec3539f762a50. Do you mean to say you think it's a separate thing or not a separate thing?